include/libdnf/transaction/transaction_item_action.hpp

```diff
@@ -28,19 +28,15 @@ namespace libdnf::transaction {
 // Any time you add a new action, change functions that resolve reasons,
 // because removed items (RPMs) must be excluded from reason resolution:
 // * Package.cpp - Package::resolveTransactionItemReason
+// TODO(jmracek) Copy enum to SWDB including their string definitions and already removed values to keep compatibility
 enum class TransactionItemAction : int {
     INSTALL = 1,        // a new package that was installed on the system
     DOWNGRADE = 2,      // an older package version that replaced previously installed version
-    DOWNGRADED = 3,     // an original package version that was replaced
-    // TODO(jmracek) OBSOLETE action is not used in DNF-4 and DNF-5. What about to remove it?
-    OBSOLETE = 4,       //
-    OBSOLETED = 5,      //
     UPGRADE = 6,        //
-    UPGRADED = 7,       //
     REMOVE = 8,         // a package that was removed from the system
     REINSTALL = 9,      // a package that was reinstalled with the identical version
-    REINSTALLED = 10,   // a package that was reinstalled with the identical version (old repo, for example)
-    REASON_CHANGE = 11  // a package was kept on the system but it's reason has changed
+    REASON_CHANGE = 11, // a package was kept on the system but it's reason has changed
+    REPLACED = 12       // Replace actions UPGRADED, DOWNGRADED, REINSTALLED, OBSOLETED
 };
```

Comment:
We'll probably need (just a detail, to avoid possible gaps in the numbering):
REASON_CHANGE_FROM=12
REPLACED=13

I'm not sure about removing the items from the enum; We'll probably figure out this when dealing with database migrations and other stuff.

Reply:
I've got a suggestion to include REASON_CHANGE_FROM when we will need it. Therefore I remove it from PR.